doc/tutorials/highgui/trackbar/trackbar.markdown

```diff
@@ -88,10 +88,10 @@ Explanation
 
 We only analyze the code that is related to Trackbar:
 
--#  First, we load 02 images, which are going to be blended.
+-#  First, we load two images, which are going to be blended.
     @code{.cpp}
-    src1 = imread("../../images/LinuxLogo.jpg");
-    src2 = imread("../../images/WindowsLogo.jpg");
+    src1 = imread("../data/LinuxLogo.jpg");
+    src2 = imread("../data/WindowsLogo.jpg");
     @endcode
```

Comment:
indeed i checked before if possible to replace all @code by @snippet but there are too many @code.
i choosed to update some @code part if the sample has updated. for now it is an overwhelmed work for me to replace all the @code by @snipped